modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -81,4 +81,10 @@
             <class name="org.wso2.am.integration.tests.application.ApplicationConsumerSecretRegenerateTestCase"/>
         </classes>
     </test>
+
+    <test name="apim-integration-tests-api-lifecycle" preserve-order="true" parallel="false">
```

Comment:
We are planning to include all the tests at once in the testng file. Shall we remove this for now?

Reply:
Fixed